packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -165,32 +200,35 @@ export default Component.extend({
   /**
    * @property {Object} groupedData - data grouped by grouping column specified in selectedSubtotal
    */
-  groupedData: computed('selectedSubtotal', 'rawData', function () {
+  groupedData: computed('selectedSubtotal', 'rawData', function() {
     let groupingColumn = get(this, 'selectedSubtotal'),
-        rawData = get(this, 'rawData');
+      rawData = get(this, 'rawData');
 
     if (groupingColumn !== 'dateTime') {
       groupingColumn = `${groupingColumn}|id`;
     }
 
-    return groupBy(rawData,
-      row => get(row, groupingColumn)
-    );
+    return groupBy(rawData, row => get(row, groupingColumn));
   }),
 
   /**
    * @property {Object} tableData
    */
-  tableData: computed('rawData', 'columns', 'options.showTotals.{grandTotal,subtotal}', function () {
-    let tableData = this._computeSubtotals(),
+  tableData: computed(
+    'rawData',
+    'columns',
+    'options.showTotals.{grandTotal,subtotal}',
+    function() {
+      let tableData = this._computeSubtotals(),
         rawData = get(this, 'rawData');
 
-    if (!get(this, 'options.showTotals.grandTotal')) {
-      return tableData;
-    }
+      if (!get(this, 'options.showTotals.grandTotal')) {
+        return tableData;
+      }
 
-    return [...tableData, this._computeTotal(rawData, 'grandTotal')];
-  }),
+      return [...tableData, this._computeTotal(rawData, 'grandTotal')];
+    }
+  ),
```

Comment:
Can we revert these changes?

Reply:
I think prettier made these changes

Reply:
yeah, prettier... haven't changed anything here